Frontend/library/src/Inputs/GamepadController.ts

```diff
@@ -65,31 +88,29 @@ export class GamePadController {
         }
     }
```

Comment:
Just for sanity, can you confirm streaming and using a gamepad to move around etc is still working?